cloudfront/wellcomelibrary.org/edge-lambda/src/catalogue.ts

```diff
@@ -0,0 +1,14 @@
+export type Work = {
+    type: 'Work' | 'Collection' | 'Section' | 'Series';
+    id: string;
+    title: string;
+};
+
+export type CatalogueResultsList<Result = Work> = {
+    type: 'ResultList';
+    totalResults: number;
+    results: Result[];
+    pageSize: number;
+    prevPage: string | null;
```

Comment:
This is `string | undefined`, as is `nextPage`

Reply:
Literally copied out of https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/model/catalogue.ts#L299